### PR TITLE
Update search bar behavior; Rename top level menu items; Fix bug assigning inspectors

### PR DESCRIPTION
**Pull request by nayr974**

### CODE REVIEW

File: services/core-web/src/components/common/RenderGroupedSelect.js [hunk 1]
```diff
@@ -50,10 +51,6 @@ const RenderGroupedSelect = (props) => (
       showSearch
       placeholder={props.placeholder}
       optionFilterProp="children"
-      filterOption={(input, option) =>
```
Comment (JustinMacaulay) [inline services/core-web/src/components/common/RenderGroupedSelect.js:53]: Are these the changes that fixed the issue with the inspector selection? Also is there no filtering needed on this common component for other sections of the application where it's used?

**Review by JustinMacaulay — COMMENTED** [review]
I just had the one comment but everything else looks good to me.

File: services/core-web/src/components/common/RenderGroupedSelect.js (hunk 1 above)
Comment (nayr974) [reply services/core-web/src/components/common/RenderGroupedSelect.js:53]: Yes. The filtering happens natively so I don't know if this is a workaround that was needed but everything works as expected now.

Comment (JustinMacaulay) [reply services/core-web/src/components/common/RenderGroupedSelect.js:53]: Sounds good!

**Review by JustinMacaulay — APPROVED** [review]
All looks good to me!

**Comment by Jsyro** [comment]
Looks awesome, Ryan. 

There is an existing bug that i wanted to raise. if you type in the quick search bar, then try and click to move the cursor, the field gets cleared, doesn't have to be addressed here, but it might be a quick fix.

**Review by Jsyro — APPROVED** [review]
Thanks for fixing the onclick.

**Pull request merged**